### DISCUSSION TITLE
Spelldex - Android app to browse and search spells

### DISCUSSION THREAD

**Discussion by palak-sethi** [danielschuster-muc/potter-db#824] [Show and tell]
Hi everyone,

Forgot your spells in the middle of a duel or before your OWL exams? Use Spelldex to brush up on your Charms.
 
Spelldex is a comprehensive Android app designed to provide a curated list of spells, featuring paging for smooth navigation, Room database integration for efficient caching, and a powerful search functionality.

Check out the project : https://github.com/palak-sethi/Spelldex

![spelldex](https://github.com/danielschuster-muc/potter-db/assets/51605219/7071e153-7265-411d-8a00-7dc7b603fcd2)

**Comment by danielschuster-muc** [comment]
That’s really cool. Nice to see others creating something with the PotterDB.
